static/js/components/forms/RegisterEmailForm.js

```diff
@@ -29,25 +29,31 @@ const RegisterEmailForm = ({ onSubmit }: Props) => (
       email:     "",
       recaptcha: SETTINGS.recaptchaKey ? "" : undefined
     }}
-    render={({ isSubmitting, isValid, setFieldValue }) => (
+    render={({ isSubmitting, setFieldValue }) => (
       <Form>
-        <label htmlFor="email">
-          Email
-          <Field type="email" name="email" />
-        </label>
-        <ErrorMessage name="email" component="div" />
+        <div className="form-group">
+          <label htmlFor="email">Email</label>
+          <Field name="email" className="form-control" component={EmailInput} />
+          <ErrorMessage name="email" component={FormError} />
+        </div>
         {SETTINGS.recaptchaKey ? (
-          <React.Fragment>
+          <div className="form-group">
             <ScaledRecaptcha
               onRecaptcha={value => setFieldValue("recaptcha", value)}
               recaptchaKey={SETTINGS.recaptchaKey}
             />
             <ErrorMessage name="recaptcha" component="div" />
```

Comment:
Could you add `component={FormError}` to the recaptcha error so it gets the same treatment?